### DISCUSSION TITLE
Approved Account Yet No Rewards

### DISCUSSION THREAD

**Discussion by JewelPaul** [community/community#140551] [GitHub Education]
### Hi everyone,

Hello,

I recently got verified and approved for github student developer pack, but despite 3 days passing since the approval, I have not received any of the rewards or benefits that were promised.

Details:
Date of Verification: 1st october 2024
Account Status: Verified and Approved
Issue: No rewards or benefits received
Expected Outcome: I should have access to the rewards and benefits within a reasonable time after verification.
I’ve double-checked my account status, and everything appears to be in order. I’ve also followed any steps outlined in the documentation, but there seems to be a delay.

Steps Taken:
Verified and approved account status.
Waited 3 days for benefits to be applied.
Checked the documentation and FAQs for potential delays or additional actions required.
Request:
Could someone from the support or development team please help look into this issue? Is there any known delay or issue affecting reward distribution for verified users?

Any assistance or guidance would be greatly appreciated. Thank you in advance for your help!

Best regards,
Jewel Gabriel Paul
JewelPaul- github username

**Comment by davevad93** [comment]
Hi @JewelPaul ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @JewelPaul was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by JewelPaul** [reply]
Thank you so much github

**Reply by davevad93** [reply]
I'm not part of the GitHub team, but you're welcome @JewelPaul! 😄👍
